maven/pom-base.xml

```diff
@@ -12,7 +12,6 @@
   <groupId>__GROUP_ID__</groupId>
   <artifactId>__ARTIFACT_ID__</artifactId>
   <version>__VERSION__</version>
-  <packaging>jar</packaging>
```

Comment:
OOC why did you remove this ? The default value is `pom` right ?

Reply:
No, the default is jar, you need to specify it for pom/war/etc, but it need not be specified for jar.

https://maven.apache.org/pom.html#Packaging

> When no packaging is declared, Maven assumes the packaging is the default: `jar`.

Reply:
and you don't inherit the one from the parent pom ?

Reply:
Correct. That wouldn't make any sense, since all poms, even without an explicit parent, have a parent pom that defines plugin versions, etc, and no project could ever take advantage of that default if it used a parent pom (if packaging isn't "pom", you can't inherit from it). https://maven.apache.org/pom.html#inheritance

Reply:
If you'd rather I revert this line, I'm happy to do so, there's no problem with being explicit about defaults. It just feels a little like calling object.tostring() when already concatenating with other strings, it isn't wrong, its just verbose.

Reply:
nope I'm fine, I need to do that for the other projects in order to be consistent